packages/hydrogen/src/seo/generate-seo-tags.test.ts

```diff
@@ -172,6 +181,22 @@ describe('generateSeoTags', () => {
         ]
       `);
     });
+
+    it('should warn if the title is too long', () => {
+      // Given
+      const input = {
+        title: 'Snowdevil'.padEnd(121, '.'), // 121 characters
+      };
+
+      // When
+      generateSeoTags(input);
+
+      // Then
+
+      expect(console.warn).toHaveBeenCalledWith(
+        'Error in SEO input: `title` should not be longer than 120 characters',
```

Comment:
Its a runtime value so it would be hard to know what they have here without running the app/requesting the data from the admin. I don't disagree, I think there is a future project where we provide proper dev-tooling for this sort of auditing. Let's talk more about it down the road. 💯